.github/workflows/test-integrations.yml

```diff
@@ -378,10 +386,14 @@ jobs:
         run: |
           cd ./test/integration/consul-container/test/upgrade
           NUM_RUNNERS=$TOTAL_RUNNERS
-          NUM_DIRS=$(go test ./...  -list=. -json | jq -r '.Output | select (. !=null) |  select(. | startswith("Test")) | gsub("[\\n\\t]"; "")' | wc -l)
+          NUM_DIRS=$(find ./test -maxdepth 2 -type d | wc -l)
```

Comment:
This one looks like a completely different command?

Reply:
ahh, yikes!  ty!